docs/aws-deployment.md

```diff
@@ -42,9 +41,22 @@ platform for running institutional analytics.
     $ export PATH=~/Library/Python/2.7/bin:$PATH
   ```
 
-  For more detais on install eb refer:
+  For more details on install eb refer:
   http://docs.aws.amazon.com/elasticbeanstalk/latest/dg/eb-cli3-install.html
 
+## 3. Pre-deployment set up
+
+  ### Steps
+  1. Create dedicated S3 buckets.
+  2. Enable transfer acceleration.
+  3. Enable encryption using KMS keys.
+  4. Setup Lifecycle rules to retain only last 15 days of data in the the bucket/canvas-data/daily/*
+  5. Set up a single node Redshift cluster with firewalls
```

Comment:
@sandeepmjay Nit-picky: "setup" vs. "set up".  We want the latter for the above.